lib/actions/ssh.ts

```diff
@@ -259,22 +257,16 @@ export const ssh: CommandDefinition<
 		}
 
 		// this will be a tunnelled SSH connection...
-		await exitIfNotLoggedIn();
+		await checkLoggedIn();
 		const uuid = await getOnlineTargetUuid(sdk, applicationOrDevice);
 		let version: string | undefined;
 		let id: number | undefined;
 
-		try {
-			const device = await sdk.models.device.get(uuid, {
-				$select: ['id', 'supervisor_version', 'is_online'],
-			});
-			id = device.id;
-			version = device.supervisor_version;
-		} catch (e) {
-			if (e instanceof BalenaDeviceNotFound) {
-				exitWithExpectedError(`Could not find device: ${uuid}`);
-			}
-		}
```

Comment:
I found that this `try/catch` block was potentially hiding errors other than `BalenaDeviceNotFound`. And for `BalenaDeviceNotFound` itself, it is already handled in `errors.ts` and can be allowed to bubble up:
```
$ ./bin/balena ssh whatwasthat
BalenaDeviceNotFound: Device not found: whatwasthat
```